src/OrchardCore.Modules/OrchardCore.Menu/Views/MenuItemLink-ContentMenuItem.cshtml

```diff
@@ -8,6 +7,10 @@
     var routeValues = new RouteValueDictionary(AutorouteOptions.Value.GlobalRouteValues);
     routeValues[AutorouteOptions.Value.ContentItemIdKey] = menuItemPart.ContentItem.Content.ContentMenuItemPart.SelectedContentItem.ContentItemIds[0];
     var linkUrl = Url.RouteUrl(routeValues);
+
+    TagBuilder tag = Tag(Model, "a");
+    tag.Attributes["href"] = linkUrl;
+    tag.InnerHtml.Append((string)menuItemPart.Name);
 }
 
-<a href="@linkUrl">@menuItemPart.Name</a>
+@tag
```

Comment:
Don't remove to EOL at the end of the files

Reply:
Put it back in.